unlock-app/src/components/interface/svg/template.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * https://www.smooth-code.com/open-source/svgr/docs/cli/#use-a-specific-template
+ * Template must return a Babel AST, the template function take three arguments:
+ * @param {*} api: API methods provided by Babel
+ * @param {*} opts: SVGR options
+ * @param {*} values: Pre-computed values to use (or not) in your templates
+ */
+function svgTemplate(
+  { template },
+  opts,
+  { imports, componentName, props, jsx, exports }
+) {
+  return template.ast`
+    ${imports}
+    import PropTypes from 'prop-types'
+    const ${componentName} = ${props} => ${jsx}
+
+    ${componentName}.propTypes = {
+      title: PropTypes.string
+    }
+
+    ${componentName}.defaultProps = {
+      title: ''
```

Comment:
have you tried using null? This might keep the snapshots the same (using `<title />`)

Reply:
I have not... but I remember that when we use type checking in another project it would complain about `null` because null is not a string and we had to replace all of the types with '<type> or null' which was painful.